wal/src/component/scheduler.rs

```diff
@@ -106,7 +106,7 @@ pub struct Scheduler {
 }
 
 impl Scheduler {
-    fn new() -> Self {
+    pub fn new() -> Self {
```

Comment:
czemu? może wogóle to powinien być default a nie new tak mnie naszło

Reply:
default zrobiony